src/content/devices/input-output/js/test.js

```diff
@@ -15,123 +15,123 @@ var test = require('tape');
 var webdriver = require('selenium-webdriver');
 var seleniumHelpers = require('../../../../../test/selenium-lib');
 
-test('Fake device selection and check video tag dimensions in ' +
-    'input-output demo', function(t) {
-  // FIXME: use env[SELENIUM_BROWSER] instead?
-  var driver = seleniumHelpers.buildDriver();
+test('Fake device selection and check video element dimensions '+
```

Comment:
Yes — this is a nit. 

I don't think this should be allowed — I'll check if there's a rule available.